### DISCUSSION TITLE
Projections for embedded documents: Path collision error

### DISCUSSION THREAD

**Discussion by awesomelike** [Automattic/mongoose#11805] [Q&A]
I am using projections similar to those described here: https://stackoverflow.com/questions/64059795/mongodb-get-error-message-mongoerror-path-collision-at-activity

I am getting the exact same error. Is there any way to achieve this type of projection in Mongoose with MongoDB >=4.4?

**Comment by vkarpov15** [comment]
@awesomelike please open an issue and follow the issue template. Without code samples and version numbers, hard for us to help.
